relayer/cmd/run/parachain/command.go

```diff
@@ -14,17 +14,19 @@ import (
 	"github.com/mitchellh/mapstructure"
 	"github.com/sirupsen/logrus"
 	"github.com/snowfork/snowbridge/relayer/chain/ethereum"
+	para "github.com/snowfork/snowbridge/relayer/chain/parachain"
 	"github.com/snowfork/snowbridge/relayer/relays/parachain"
 	"github.com/spf13/cobra"
 	"github.com/spf13/viper"
 	"golang.org/x/sync/errgroup"
 )
 
 var (
-	configFile     string
-	privateKey     string
-	privateKeyFile string
-	privateKeyID   string
+	configFile          string
+	privateKey          string
+	privateKeyFile      string
+	privateKeyID        string
+	parachainPrivateKey string
```

Comment:
@yrong do we still need the substrate private key for the parachain relayer? Maybe for the delivery proofs? I don't see it in your [Parachain Relayer V2 PR](https://github.com/Snowfork/snowbridge/pull/1321) anymore, but perhaps it was in the v2 branch already.

Reason why I am asking is I want to add the key to the start-relayer scripts but just want to make sure it is still necessary.

Reply:
Yeah, we need that for submitting delivery proofs to BH. 

I wrap a script specific for the V2 message relayer in https://github.com/Snowfork/snowbridge/blob/v2/web/packages/test/scripts/start-relayer-v2.sh and test by running a legacy V1 relayer side by side with the V2 message relayer.

In that way, I can test the V1 smoke tests still work as expected for the V2 setup.

Anyway, feel free to add that to start-relayer scripts.